ntbs-service/Pages/ValidationModel.cs

```diff
@@ -39,8 +39,11 @@ private void SetProperty(object model, string key, object value)
             var property = model.GetType().GetProperty(key);
 
             var converter = TypeDescriptor.GetConverter(property.PropertyType);
-            value = converter.ConvertFromString((string) value);
-
+            try
+            {
+                value = converter.ConvertFrom(value);
+            }
+            catch {}

```

Comment:
Feels like we should have something here, or an explanation why it's OK to let things fall through?